### PR TITLE
FISSA installation

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
         'caiman': ['h5py==3.3.0',
                    'scipy==1.7.0',
                    'caiman@git+https://github.com/flatironinstitute/CaImAn.git@v1.8.9'],
+	'imgconverter': [tifffile, nd2]
```
Comment (kabilar) [inline setup.py]: I would suggest locking to the version that was tested.  Thanks.

**Comment by kabilar** [comment]
Thanks @tdincer.  Are you planning on using `tifffile` or `scanreader` to read in the output of this function?  If `scanreader`, did you happen to test if it loads correctly?

**Comment by tdincer** [comment]
@kabilar When I used `scanreader`, it didn't read because the tif file lacks some header information. So, I'll use `tifffile`.

Here is the actual complaint: `ScanImageVersionError('Could not find ScanImage version in the tiff header')`

File: setup.py (hunk 1 above)
Comment (tdincer) [reply setup.py]: @kabilar Thank you! Done.

**Comment by kabilar** [comment]
Thanks for testing, @tdincer.  Perhaps we should add an `Outputs` section in the docstring and make note that this file should be read in with `tifffile` and not `scanreader`, since we quite often use `scanreader`.

File: element_interface/image_converter.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import nd2
+import tifffile
+
+
+def nd2totif(nd2_file):
```
Comment (kabilar) [inline element_interface/image_converter.py]:
```suggestion
def nd2_to_tif(nd2_file):
```

File: setup.py [hunk 3]
```diff
@@ -41,6 +41,7 @@
         'caiman': ['h5py==3.3.0',
                    'scipy==1.7.0',
                    'caiman@git+https://github.com/flatironinstitute/CaImAn.git@v1.8.9'],
+	'imgconverter': ['tifffile==2021.11.2', 'nd2==0.1.6']
```
Comment (kabilar) [inline setup.py]:
```suggestion
	'image_converter': ['tifffile==2021.11.2', 'nd2==0.1.6']
```

**Comment by tdincer** [comment]
@kabilar Let's not proceed with this PR as our suite2p fork supports `nd2` images now. On the other hand, we may need this converter for caiman in the future.

**Review by dimitri-yatsenko — APPROVED** [review]

File: setup.py [hunk 4]
```diff
@@ -41,6 +41,8 @@
         'caiman': ['h5py==3.3.0',
                    'scipy==1.7.0',
                    'caiman@git+https://github.com/flatironinstitute/CaImAn.git@v1.8.9'],
+        'fissa': ['fissa@https://github.com/rochefort-lab/fissa.git'],
```
Comment (kabilar) [inline setup.py]:
```suggestion
        'fissa': ['fissa@git+https://github.com/rochefort-lab/fissa.git'],
```

Comment (kabilar) [reply setup.py]: Hi @tdincer, is the `git+` prefix required?  In my previous testing, I believe it is required but I could be mistaken.

**Review by kabilar — APPROVED** [review]
Thanks @tdincer!  Should we version lock version `FISSA`?  If not, I can merge.

**Comment by tdincer** [comment]
FISSA's last release is buggy. Let's not version lock until they release a new one.

File: setup.py (hunk 4 above)
Comment (tdincer) [reply setup.py]: You are right `git+` prefix is required.

**Comment by tdincer** [comment]
Ready to merge!

**Pull request merged**